example-app/SantokuApp/src/screens/demo/push-notification/usePushNotification.tsx

```diff
@@ -0,0 +1,106 @@
+import messaging from '@react-native-firebase/messaging';
+import axios, {AxiosError} from 'axios';
+import {useCallback, useState} from 'react';
+
+import {accountApi, AppConfig} from '../../../framework';
+import {ApiResponseError} from '../../../framework/backend';
+import {ErrorResponse} from '../../../generated/api';
+
+export const usePushNotification = () => {
+  const [authStatus, setAuthStatus] = useState<string>();
```

Comment:
`AuthStatusType`  を定義したほうがより良いかと思います。

```suggestion
type AuthStatusType = 'NOT_DETERMINED' | 'DENIED' | 'AUTHORIZED' | 'PROVISIONAL';

export const usePushNotification = () => {
  const [authStatus, setAuthStatus] = useState<AuthStatusType >();
```